doc/ImplementingApplications/ImplementingApplication/ConceptOfLogDirection/ConceptOfLogDirection.md

```diff
@@ -0,0 +1,48 @@
+## Directing Logs and Traces
+
+In all the approved applications of ApplicationPattern , 
+* the OAM layer logs the OAM request information to a Tiny Application Controller called “OAMLog”(OL)
+* Both the Individual and Basic services log the service request to a Tiny Application Controller called “ExecutionAndTraceLog”(EATL)
+
+### OAM Request
+
+ ![OAMLogFlow](Images/oamLogFlow.png)
+After processing an OAM request , the following details will be logged into the OL application using the service “record-oam-request” , 
+- application-name : Name of application addressed by OaM request
+- release-number : Release number of application addressed by OaM request
+- method : HTTP method of the request (for example GET, PUT etc.,)
+- resource : 'Resource addressed'
+- stringified-body : Body of the OaM request
+- response-code : Response code on the OaM request
+- user-name : Username extracted from Authorization header used to identify in basicAuth
+- timestamp : Date and time when response was sent
+
+The logged information shall be useful to track the configuration changes. 
+OL provided the following services to track the configuration changes , 
+* list-records
+* list-records-of-application
+
+### Basic and Individual Service Request
+
+ ![EATLFlow](Images/EATLFlow.png)
```

Comment:
EATLFlow.png should be changed

Reply:
Many thanks. Updated the picture of EATLFlow.png